packages/grid/demo/stories/data.ts

```diff
@@ -50,3 +50,64 @@ export const GRID_ROWS: IGridRow[] = [
+  `Farro platter peanut butter Italian pepperoncini edamame hummus sweet potato green onions winter picnic salad artichoke hearts cauliflower crunchy. Shiitake mushrooms shallots veggie burgers garlic sriracha noodles green grapes chocolate green pepper coconut sugar tomato and basil habanero golden figs cumin cilantro lime vinaigrette blackberries ultra creamy avocado pesto.`,
+
+  `Ghost pepper hemp seeds lemongrass black beans peaches sparkling pomegranate punch avocado dressing drizzle spring candy cane winter avocado avocado basil pesto creamy cauliflower alfredo. Chocolate cookie thyme chili raspberries cool cucumbers zesty tofu pad thai portobello mushrooms red lentil curry a delicious meal vitamin glow crumbled lentils blueberry pops strawberry mango smoothie.
+
+Lemon tahini dressing hazelnut shiitake spiced pumpkin chili Thai dragon pepper udon noodles bento box broccoli soba noodles grapefruit roasted peanuts lemon lime minty Thai curry sriracha pecans raspberry fizz summertime mediterranean tempeh green papaya salad casserole asian pear Thai sun pepper balsamic vinaigrette jalapeño simmer alfalfa sprouts. Cayenne bite sized instant pot lemon citrusy cool off sweet potato cherry bomb pepper onion matcha dragon fruit fresh Bolivian rainbow pepper smoky maple tempeh glaze salad dark chocolate.
+
+Chinese five-spice powder dill sandwiches pumpkin morning smoothie bowl naga viper potato arugula salad homemade balsamic grains cozy cinnamon oatmeal sleepy morning tea kale caesar salad earl grey latte roasted butternut squash. Delightful blueberry scones coconut milk red pepper ginger lemongrass agave green tea creamiest summer fruit salad bruschetta Mexican fiesta cherries orange muffins frosted gingerbread bites.`,
+  `Green tea ginger carrot spiced juice sesame soba noodles coconut tahini drizzle butternut mix refreshing cucumber splash apple vinaigrette entree Bulgarian carrot pomegranate coriander cinnamon Sicilian pistachio pesto golden cayenne pepper kimchi scotch bonnet pepper chili pepper kale sweet potato black bean burrito roasted brussel sprouts Thai super chili cashew with edamame. Apricot fig arugula cashew salad basmati apples Caribbean red habanero lentils green bowl blueberries salted tabasco pepper almond milk crispy iceberg lettuce mediterranean vegetables ginger tofu miso turmeric glazed aubergine heat guacamole appetizer cranberry spritzer pineapple salsa.`
+];
+
+export const SIMPLE_SPLITTER_PANES: ISplitterPane[] = [
+  {
+    name: 'Pane 1',
+    splitters: [
+      {
+        layoutKey: 'row-1',
+        orientation: 'bottom',
+        min: 0,
+        max: 2
+      },
+      {
+        layoutKey: 'column-1',
+        orientation: 'end',
+        min: 0,
+        max: 2
+      }
```

Comment:
[nit] if possible (and I'm sure it is) let's rearrange these splitters so they present in clockwise tab order, starting from the top. If it helps make the case, this follows the precedent set by CSS shorthand (`border`, `margin`, `padding`, etc) – moving from `top`, `right`, `bottom`, `left`.

Reply:
Talked offline and found out the DOM doesnt behave like this naturally.